teamtalk/instance.py

```diff
@@ -72,6 +72,7 @@ def connect(self) -> bool:
             return False
         result, msg = _waitForEvent(self.super, sdk.ClientEvent.CLIENTEVENT_CON_SUCCESS)
         if not result:
+            self.bot.dispatch("my_connection_lost", self.server)
```

Comment:
This is not really how the event should be used. I'm thinking that, if we don't have a connection to begin with, we can't lose it. Aka, we can listen for it in the process message call and dispatch it that way, but shouldn't dispatch it during connection.